bin/_mkproject.bat

```diff
@@ -1,29 +1,39 @@
-:: Project API
+::  Project API
 ::
-:: This file is called whenever a user enters a project, such p999_Meindbender_Sync
+::  This file is called whenever a user enters a project, such p999_Meindbender_Sync
 ::
-:: Arguments:
-::   %1: Path = Absolute path to projects
-:: 	 %2: Name = Name of project
-:: 	 %3: Silo = Parent directory name of assets
+::  Arguments:
+::    %1: Path = Absolute path to projects
+:: 	  %2: Name = Name of project
+::    %3: \f02_prod
+::    %4: Silo = Parent directory name of assets
 ::
-:: Variables:
-::   %PROJECTDIR%: Absolute path to project directory
+::  Variables set by bat:
+::    %MINDBENDER_SETASSETCATEGORY%: Is the currently chosen category of assets
```

Comment:
it is a reset for the variable, it was suppose to be integrated with the extra step option bat that would allow the user to chose the category.